strawberryfields/gbs/vibronic.py

```diff
@@ -172,7 +99,7 @@ def gbs_params(
         w (array): normal mode frequencies of the initial electronic state (:math:`\mbox{cm}^{-1}`)
         wp (array): normal mode frequencies of the final electronic state (:math:`\mbox{cm}^{-1}`)
         Ud (array): Duschinsky matrix
-        d (array): Duschinsky displacement vector corrected with wp
+        d (array): Duschinsky displacement vector
```

Comment:
The term d here is actually delta which is related to Duschinsky displacement d as:
delta = sqrt(w / h-bar) * d, with w being the angular frequency. We should probably explain it somewhere.